src/dmd/globals.h

```diff
@@ -122,6 +122,7 @@ struct Param
     bool useInline;     // inline expand functions
     bool useDIP25;      // implement http://wiki.dlang.org/DIP25
     bool noDIP25;       // revert to pre-DIP25 behavior
+    bool useDIP1021;        // implement http://wiki.dlang.org/DIP1021
```

Comment:
That page does not exist on the wiki.

Reply:
It will.

Reply:
I believe you meant this URL: https://github.com/dlang/DIPs/blob/master/DIPs/DIP1021.md